website/grading.rst

```diff
@@ -3,92 +3,128 @@
 Grading
 =======
 
-.. warning::
+Like CMSC 14200, we will be using a form of grading known as *specifications grading*.
+The goal of specifications grading is to help students focus on their mastery of the
+material and to identify areas for improvement as the quarter progresses. You will
+get the most out of this scheme if you focus on skills, not on scores.
+
+All work in this course will be evaluated using a E/S/N/U scale:
 
+- **Excellent** (E): The work is either perfect, or has only very minor flaws. In some coursework,
+  achieving an E will require completing extra work above and beyond what is required to earn an S.
+- **Satisfactory** (S): The student has put in a good-faith effort to complete all the work
+  and demonstrated sufficient mastery of the material.
+- **Needs Improvement** (N): The student has put in a good-faith effort to complete the work,
+  but revealed a lack of mastery in the material that can be addressed via concrete feedback.
+  The work could become Satisfactory, or even Excellent, with revisions.
+- **Unsatisfactory** (U): The student did not make a good-faith effort to complete the work.
+  For example, they submitted no work at all, or some token amount of work. Some coursework
+  specify a specific minimum amount of work to complete to avoid a U.
```

Comment:
Could italicize the last sentence here about the specified minimum, since it's somewhat distinct from the example that came before it. So someone speedreading might not actually catch it.